src/machine/board_arduino.go

```diff
@@ -1,15 +1,7 @@
-// +build arduino
+// +build arduino,!atmega2560
 
 package machine
 
-// Return the current CPU frequency in hertz.
-func CPUFrequency() uint32 {
-	return 16000000
-}
-
-// LED on the Arduino
-const LED Pin = 13
-
```

Comment:
This appears to be leftover from testing other changes?

Reply:
believe it is actually.  I was trying to fix the CircleCI wasm build.